src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -38,6 +41,23 @@ public override JObject ToJson()
 
         public override bool Verify(DataCache snapshot, Transaction tx)
         {
+            // Check already stored signers
+
+            var conflictingSigners = tx.Signers.Select(s => s.Account);
+
+            foreach (var attr in tx.GetAttributes<Conflicts>())
+            {
+                var conflictRecord = snapshot.TryGet(
+                    new KeyBuilder(NativeContract.Ledger.Id, LedgerContract.Prefix_Transaction).Add(attr.Hash))?
+                    .GetInteroperable<TransactionState>();
+                if (conflictRecord == null) continue;
```

Comment:
```suggestion
                if (conflictRecord is null || conflictRecord.Transaction is not null) continue;
```

Just to be more precise.